### DISCUSSION TITLE
can't build in vs2022 17.3

### DISCUSSION THREAD

**Discussion by pedoc** [winsiderss/systeminformer#2192] [Questions / Answers] [answered]
The latest commit (47779f2c) cannot be compiled in vs2022 17.3 version
windows sdk 10.0.22621.0
vc:v143

![image](https://github.com/user-attachments/assets/25f13497-7d9b-4ae9-9253-162258948704)

It looks like the error is in the Windows SDK. Has anyone encountered this before?

**Comment by JustinJayL** [comment, marked as answer]
I just ran into this same issue, the problem is that you have an outdated Windows 11 SDK. You need to install the latest version, and then re run build_init and then build_release.

**Reply by pedoc** [reply]
Due to issues with driver signatures, I have stopped compiling myself and now I use the canary version from the official website directly.
